helm/robusta/templates/runner-service-account.yaml

```diff
@@ -263,6 +263,12 @@ kind: ServiceAccount
 metadata:
   name: {{ .Release.Name }}-runner-service-account
   namespace: {{ .Release.Namespace }}
+  {{- if .Values.runner.serviceAccount.annotations }}
```

Comment:
can you move the annotations to the `runnerServiceAccount` helm value? (see line 272 below)